website/docs/content/writing-guidelines/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Writing Guidelines
+caption: Guidelines for writing consistent UI copy.
+description: Guidelines for writing consistent UI copy.
+previewImage: assets/illustrations/content/writing-guidelines.jpg
+---
+
+## Date and time
```

Comment:
[blocking] @Dhaulagiri mentioned this in the chat during today's Leads meeting, but it's strange that there's no reference or cross-link to the Time component. That needs to be present.